code/modules/projectiles/guns/projectile/sniper.dm

```diff
@@ -173,7 +173,7 @@
 	icon_state = ".50"
 
 /obj/item/projectile/bullet/sniper/accelerator
-	icon_state = "gaussweak"
+	icon_state = "scatterlaser"
```

Comment:
This is the icon state for the _projectile sniper rifle round_, not your new subtype. Did you check that the heavy laser sound is preserved as well? (Unless you do not intend to keep it?)